app-modules/prospect/src/Imports/ProspectImporter.php

```diff
@@ -0,0 +1,154 @@
+                    $status = ProspectStatus::query()
+                        ->whereKey($value)
+                        ->orWhere('name', $value)
+                        ->first();
+
+                    if ($status) {
+                        return;
+                    }
+
+                    $fail('The selected status is invalid.');
+                }])
+                ->guess(['status'])
+                ->fillRecordUsing(function (Prospect $record, string $state) {
+                    $status = ProspectStatus::query()
+                        ->where('name', $state)
+                        ->first();
+
+                    if (! $status) {
+                        return;
+                    }
+
+                    $record->status()->associate($status);
+                })
+                ->requiredMapping(),
+            ImportColumn::make('source_id')
+                ->label('Source')
+                ->rules([function (string $attribute, mixed $value, Closure $fail) {
+                    $source = ProspectSource::query()
+                        ->whereKey($value)
+                        ->orWhere('name', $value)
+                        ->first();
+
+                    if ($source) {
+                        return;
+                    }
+
+                    $fail('The selected source is invalid.');
+                }])
+                ->guess(['source'])
+                ->fillRecordUsing(function (Prospect $record, string $state) {
+                    $source = ProspectSource::query()
+                        ->where('name', $state)
+                        ->first();
+
+                    if (! $source) {
+                        return;
+                    }
+
+                    $record->source()->associate($source);
+                })
+                ->requiredMapping(),
+            ImportColumn::make('description'),
+            ImportColumn::make('email')
+                ->rules(['email']),
+            ImportColumn::make('email_2')
+                ->rules(['email']),
+            ImportColumn::make('mobile'),
+            ImportColumn::make('sms_opt_out')
+                ->boolean()
+                ->rules(['boolean']),
+            ImportColumn::make('email_bounce')
+                ->boolean()
+                ->rules(['boolean']),
+            ImportColumn::make('phone'),
+            ImportColumn::make('address'),
+            ImportColumn::make('address_2'),
+            ImportColumn::make('birthdate')
+                ->rules(['date']),
+            ImportColumn::make('hsgrad'),
+        ];
+    }
+
+    public function resolveRecord(): ?Model
+    {
+        $email = $this->data['email'] ?? null;
+        $email2 = $this->data['email_2'] ?? null;
+
+        $emails = array_filter(
+            [$email, $email2],
+            fn (mixed $state): bool => filled($state),
+        );
+
+        if (empty($emails)) {
+            return new Prospect();
+        }
```

Comment:
I agree, at this point I think we can move forward with email being required. We can make the change in the DB soon as well.

Without an email engagements and other functionality just doesn't work, it would not make sense to not have an email.